src/util/functions.ts

```diff
@@ -216,3 +217,44 @@ export function checkString(content: string): CensorCheckResponse | undefined {
 		if (match) return { rule, match };
 	}
 }
+
+export function emojiEquals(x: any, y: any) {
+	if (typeof x === 'string' && typeof y === 'string') {
+		return x === y;
+	}
+
+	if (typeof x === 'string') {
+		return x === y.name;
+	}
+
+	if (typeof y === 'string') {
+		return x.name === y;
+	}
+
+	return x.identifier === y.identifier;
+}
+
+export function getEmojiFromId(client: MinehutClient, id: string) {
+	if (/^\d+$/.test(id)) return client.emojis.cache.get(id);
+
+	return id;
+}
+
+export function findImg(msg: Message) {
```

Comment:
This function can use a more descriptive name, such as `findImgFromMessage()` or `findImgFromMsg()`